libs/block-mesh-manager/src/routes/uptime_report/report_uptime.rs

```diff
@@ -47,6 +48,18 @@ pub async fn handler(
     let user = get_user_opt_by_id(&mut transaction, &api_token.user_id)
         .await?
         .ok_or_else(|| Error::UserNotFound)?;
+
+    if let Some(Json(body)) = body {
```

Comment:
why inline the query here?
should be in a method under `database/analytics/insert_client_analytics.rs`